man/runc.8.md

```diff
@@ -39,8 +39,9 @@ value for "bundle" is the current directory.
    ps           displays the processes running inside a container
    restore      restore a container from a previous checkpoint
    resume       resumes all processes that have been previously paused
+   run          create and run a container
    spec         create a new specification file
-   start        create and run a container
+   start        start signals a created container to execute the user defined process
```

Comment:
might be better phrased as `start executes the user defined process in a created container`